src/did/resolvers/mod.rs

```diff
@@ -1,3 +1,5 @@
 mod example;
+pub(crate) mod mock;
```

Comment:
don't need to be `pub(crate)` as we re-export. Also need to be `#[cfg(test)]` to avoid compilation for non test mode